sdk/tests/conformance/extensions/webgl-multi-draw.html

```diff
@@ -1014,8 +1014,29 @@
   await compositingTestFn(multiDrawElementsInstancedWEBGL);
 }
 
+function testSideEffects() {
+  debug("")
+  debug("Testing that ANGLE_instanced_arrays is implicitly enabled")
+  const canvas = document.createElement("canvas");
+  const gl = wtu.create3DContext(canvas, undefined, 1);
+  if (!gl) {
+    testFailed('WebGL context creation failed');
+    return;
+  }
+  gl.enableVertexAttribArray(0);
+  gl.getVertexAttrib(0, 0x88FE /* VERTEX_ATTRIB_ARRAY_DIVISOR_ANGLE */);
+  wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, "divisor enum unknown");
+  const ext = gl.getExtension("WEBGL_multi_draw");
+  if (ext) {
+    debug("WEBGL_multi_draw enabled");
+    gl.getVertexAttrib(0, 0x88FE /* VERTEX_ATTRIB_ARRAY_DIVISOR_ANGLE */);
```

Comment:
Suggestion if you want to avoid the argument comment:
```
var ext = ...
shouldBe("local_VERTEX_ATTRIB_ARRAY_DIVISOR_ANGLE", "ext.VERTEX_ATTRIB_ARRAY_DIVISOR_ANGLE");
gl.getVertexAttrib(0, ext.VERTEX_ATTRIB_ARRAY_DIVISOR_ANGLE);
```